dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -206,6 +206,12 @@ http.proxyredirect.httpsPort.desc=The (secure) port to redirect to. Incoming HTT
 http.autoDecompression=Auto decompression
 http.autoDecompression.desc=Specifies whether the HTTP Channel automatically decompresses incoming request body data.
 
+http.decompressionRatioLimit=Decompression ratio limit
+http.decompressionRatioLimit.desc=Specifies the maximum ratio of decompressed to compressed request body payload. This ratio is verified as the body is read by the HTTP channel, and if the decompressionTolerance is reached, the request is ended.
```

Comment:
Change,
"Specifies the maximum ratio of decompressed to compressed request body payload. This ratio is verified as the body is read by the HTTP channel, and if the decompressionTolerance is reached, the request is ended."
to,
"Specifies the maximum ratio of decompressed to compressed request body payload. The HTTP channel reads the request body and verifies the ratio. The request ends if the decompressionTolerance is reached."

Reply:
I think we should define the decompressionTolerance, 

The request ends if the maximum number of times the HTTP channel reach this ratio sepcified by decompressionTolerance

Reply:
Made the changes above.